Source: Upstream
Upstream: Yes, applied in svn (http://xbmc.org/trac/ticket/8041)
Reason: Use mysql_config to set appropriate paths

Index: configure.in
===================================================================
--- configure.in	(revision 25429)
+++ configure.in	(arbetskopia)
@@ -326,6 +326,11 @@
 AC_CHECK_HEADER([vorbis/vorbisfile.h],, AC_MSG_ERROR($missing_library))
 AC_CHECK_HEADER([vorbis/vorbisenc.h],, AC_MSG_ERROR($missing_library))
 AC_CHECK_LIB([bz2],         [main],, AC_MSG_ERROR($missing_library))
+AC_CHECK_PROG(MYSQL_CONFIG, mysql_config, YES, NO)
+if test $MYSQL_CONFIG = YES; then
+	INCLUDES+=" `mysql_config --include`"
+	LIBS+=" `mysql_config --libs`"
+fi
 AC_CHECK_LIB([mysqlclient], [main],, AC_MSG_ERROR($missing_library))
 AC_CHECK_LIB([jpeg],        [main],, AC_MSG_ERROR($missing_library)) # check for cximage
 AC_CHECK_LIB([pthread],     [main],, AC_MSG_ERROR($missing_library))
